presc/alberginia/missclassification_visuals.py

```diff
@@ -64,14 +64,14 @@ class to use for validation of the model.
 
     elif new_classes == "which-hit":
         dataset_missclass["Miss & Class"] = np.where(
-            dataset_missclass_calc["Hit"] == False,
+            not dataset_missclass_calc["Hit"],
```

Comment:
Do you want any "not" value or specifically False?

Reply:
@alberginia ^

Reply:
@crankycoder @birdsarah Here I wanted to detect the "False" values of "Hit", because thise means that the point was labelled as misclassified. I think that "not" would potentially capture other things, right? Although I don't think there are other things apart from "True" or "False" in this dataset.

Reply:
Then `dataset_missclass_calc["Hit"] is False` is the pythonic way to be specific to `True/False` as opposed to allowing `1/0` and other things.

Reply:
Thanks! There probably will be a ton of those things.  :roll_eyes:

Reply:
All good. That's why code review is great. There's always new stuff to learn. Or in my case today: new stuff to relearn after unlearning if (thanks @crankycoder).

Reply:
Actually... I've found out now that this change breaks the function, so I'll have to revert it back to what it was originally.  :P